### PR TITLE
Fix tags with no content

### PR DESCRIPTION
**Pull request by Konicai**
Should fix tags like asking.tag which have an image and no content. The image could just be linked in the text like tryitandsee.tag though, afaik

### CODE REVIEW

File: src/main/java/org/geysermc/discordbot/tags/TagsManager.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ private static void loadTags() {
                             }
 
                             if (content.toString().isEmpty()) {
-                                GeyserBot.LOGGER.warn("Tag '" + tagName + "' has empty content! Skipping tag.");
+                                GeyserBot.LOGGER.debug("Tag '" + tagName + "' has empty content!");
```
Comment (Heath123) [inline src/main/java/org/geysermc/discordbot/tags/TagsManager.java:127]: You just changed the log message, I think you meant to remove `continue`

**Review by Heath123 — CHANGES_REQUESTED** [review]

File: src/main/java/org/geysermc/discordbot/tags/TagsManager.java (hunk 1 above)
Comment (Konicai) [reply src/main/java/org/geysermc/discordbot/tags/TagsManager.java:127]: oops ty

**Comment by rtm516** [comment]
This check should just be moved into the tag construction and then not add it if the tag is empty for certain types

**Comment by Konicai** [comment]
How about something like this?
```java
String tagType = tagData.get("type");
if (tagType == null) {
    GeyserBot.LOGGER.warn("Tag '" + tagName + "' has not defined a type! Skipping tag.");
    continue;
}
if (content.toString().isEmpty()) {
    if (tagType.equals("text")) {
        if (tagData.get("image") == null) {
            GeyserBot.LOGGER.warn("Tag '" + tagName + "' of type 'text' has no text content or image! Skipping tag.");
            continue;
        }
    } else {
        GeyserBot.LOGGER.warn("Tag '" + tagName + "' of type '" + tagType + "' can only contain text content, and it is empty! Skipping tag.");
        continue;
    }
}                          
```

**Comment by rtm516** [comment]
> How about something like this? It'll mean less duplicate code in the switch block

Not quite what I meant but should sort the issue. and it is readable, just worth adding a comment before it to state what its doing.

**Review by Heath123 — APPROVED** [review]

**Review by rtm516 — APPROVED** [review]

**Pull request merged**